R/rope.R

```diff
@@ -10,7 +10,7 @@
 #'
 #' @details Statistically, the probability of a posterior distribution of being
 #'   different from 0 does not make much sense (the probability of it being
-#'   different from a single point being infinite). Therefore, the idea
+#'   different from a single point being 100%). Therefore, the idea
```

Comment:
I think infinite is correct here

Reply:
The probability of a single point is 0, so the complementary probability to that is 100-0=100%. No?

Reply:
No, I don't think so, I think infinite is correct, or maybe undefined, but I think infinite probability is common in maths.